payments/index.md

```diff
@@ -66,6 +65,24 @@ POST https://b.paylike.io/payments
     returnUrl: URL, // optional, return to this URL for redirect-based flows
   },
 
+  // optional
+  source: {
+    holder: 'destination', // optional
+  }
+  // optional
+  destination: {
```

Comment:
> optional, length: 1..1024

shouldn't the `length` be the same as:

```
destination: {
    id: /[\x20-\x7E]{1,34}/,
    holder: {
      id: /[\x20-\x7E]{1,16}/,
      name: /[\x20-\x7E]{2,30}/,
      address: /[\x20-\x7E]{1,35}/,
      city: /[\x20-\x7E]{1,25}/,

      // for country US or CA
      state: /[A-Z]{2}/,

      country: /[A-Z]{2}/, // ISO 3166-1 alpha-2 code (e.g. US)
    },
  },
  ```

Reply:
No. Payments can, legitimately, have a holder name longer than 30 characters, however, such payments won't qualify for Visa AFT processing. Other similar "programs" with schemes may have different requirements.

Reply:
Ah, sure, this is general as opposed to VISA AFT specific